assets/base/i18n/en.json

```diff
@@ -10,6 +10,26 @@
   "about.teamEditiont0": "Team Edition",
   "about.teamEditiont1": "Enterprise Edition",
   "about.title": "About {appTitle}",
+  "accessibility.attachment_button": "attachment button",
+  "accessibility.channel_nav_bar": "channel nav bar",
```

Comment:
Is this the channel header? @enahum 

Propose "channel header" in that case

Reply:
yes, that is the channel header

Reply:
Yes, it is.  Will change as suggested.